bin/__tests__/jscodeshift-test.js

```diff
@@ -44,11 +45,23 @@ function run(args, stdin, cwd) {
 }
 
 describe('jscodeshift CLI', () => {
-  function createTempFileWith(content) {
+  function createTempFileWith(content, filename) {
     var info = temp.openSync();
+    var filePath = info.path;
     fs.writeSync(info.fd, content);
     fs.closeSync(info.fd);
-    return info.path;
+    if (filename) {
+      filePath = renameFileTo(filePath, filename);
+    }
+    return filePath;
+  }
+
+  function renameFileTo(oldPath, newFilename) {
+    var projectPath = path.dirname(oldPath);
+    var newPath = path.join(projectPath, newFilename);
+    mkdirp.sync(path.dirname(newPath));
+    fs.renameSync(oldPath, newPath);
+    return newPath;
   }
```

Comment:
Renaming files in this way prevents the `temp` module from cleaning up the files after the test suite is finished running. Then again, we weren't calling `temp.track()` or `temp.cleanupSync()` in there. We might need to do that. 

Creating temp files with known names with the `temp` module isn't straightforward, since it only allows specifying prefixes and suffixes for file and directory names as opposed to the full and complete thing. We may need to manually clean up any files that run through this method ourselves.